test/sdk/e2e/get-all-pool-keys.test.ts

```diff
@@ -0,0 +1,81 @@
+import { ONE_ALPH, web3 } from '@alephium/web3'
```

Comment:
it could be nice to include the following tests:
- query on 0 pools
- `getPoolKeys` method test that fails by going over limit by 1
- `getPoolKeys` that tests exactly the limit
- `getAllPoolKeys` that works over limit
(these 3 could work on the same data so that they run faster)